src/main/java/de/srendi/advancedperipherals/common/addons/botania/ManaFlowerIntegration.java

```diff
@@ -37,4 +37,15 @@ public final int getMana() {
     public final boolean isOnEnchantedSoil() {
         return blockEntity.overgrowth;
     }
+
+    @LuaFunction(mainThread = true)
+    public final boolean isFull() {
+        return blockEntity.getMana() >= blockEntity.getMaxMana();
+    }
+
+    @LuaFunction(mainThread = true)
+    public final boolean isEmpty() {
+        return getMana() == 0;
```

Comment:
I am not sure if you should use `blockEntity.getMana()` here instead of the LuaFunction `getMana`

Reply:
Yeah blockEntity.getMana() should be used
I didn't see that